_includes/footer.html

```diff
@@ -1,7 +1,12 @@
+{% capture name_components %}
+  {% if site.translations[site.lang].main.first_name %}{{ site.translations[site.lang].main.first_name }}{% endif %}
+  {% if site.translations[site.lang].main.middle_name %}{{ site.translations[site.lang].main.middle_name }}{% endif %}
+  {% if site.translations[site.lang].main.last_name %}{{ site.translations[site.lang].main.last_name }}{% endif %}
+{% endcapture %}
     {% if site.footer_fixed %}
     <footer class="fixed-bottom">
       <div class="container mt-0">
-        &copy; Copyright {{ site.time | date: '%Y' }} {% t main.first_name %} {% t main.middle_name %} {% t main.last_name %}. {% t main.footer_text %}
```

Comment:
Why you removed the year from the footer? I believe it is worth keeping it

Reply:
Entirely by mistake. Fixed it with commit c12f263.